boilerplate/index.ts

```diff
@@ -498,11 +498,24 @@ export function createBoilerplate(sdk: CDP) {
                         }
 
                         function createCloudStorageEvent(boilerplateEvent, remoteCloudStorageApplication) {
-                            return bOps.applications.for(remoteCloudStorageApplication.id).dataevents.create({
-                                ...boilerplateEvent,
-                                schema: JSON.stringify(boilerplateEvent.schema),
-                                purposeIds: boilerplateEvent.purposeIds
-                            });
+                            if (remoteCloudStorageApplication.name !== 'Microsoft Azure Blob') {
+                                return bOps.applications.for(remoteCloudStorageApplication.id).dataevents.create({
+                                    ...boilerplateEvent,
+                                    schema: JSON.stringify(boilerplateEvent.schema),
+                                    purposeIds: boilerplateEvent.purposeIds
+                                });
+                            } else {
+                                return bOps.applications.for(remoteCloudStorageApplication.id).dataevents.create({
+                                    ...boilerplateEvent,
+                                    schema: JSON.stringify(boilerplateEvent.schema),
+                                    purposeIds: boilerplateEvent.purposeIds,
+                                    configValues: {
```

Comment:
why is this required?

Reply:
use application configSchema

Reply:
unfortunately we cant use the config schema since it holds many irrelevant fields. This would mean sorting through the fields, which is more complex than explicitly stating what we need